src/game/puppycam2.c

```diff
@@ -84,18 +85,16 @@ struct gPCOptionStruct
 
 static const struct gPCOptionStruct gPCOptions[] = { //If the min and max are 0 and 1, then the value text is used, otherwise it's ignored.
 #ifdef WIDE
-    {/*Option Name*/ 7, /*Option Variable*/ &gConfig.widescreen,       /*Option Value Text Start*/ 0, /*Option Minimum*/ FALSE, /*Option Maximum*/ TRUE},
+    {/*Option Name*/ 5, /*Option Variable*/ &gConfig.widescreen,       /*Option Value Text Start*/ 0, /*Option Minimum*/ FALSE, /*Option Maximum*/ TRUE},
```

Comment:
What is "Option Name" supposed to be? I would either sort these to make sense or replace them with enums; this struct is a bit confusingly laid out in general.